drivers/flash/soc_flash_nrf5.c

```diff
@@ -165,8 +201,339 @@ static int nrf5_flash_init(struct device *dev)
 {
 	dev->driver_api = &flash_nrf5_api;
 
+	k_sem_init(&sem_lock, 1, 1);
+
+#if defined(CONFIG_SOC_FLASH_NRF5_RADIO_SYNC)
+	k_sem_init(&sem_sync, 0, 1);
+#endif /* CONFIG_SOC_FLASH_NRF5_RADIO_SYNC */
+
 	return 0;
 }
 
 DEVICE_INIT(nrf5_flash, CONFIG_SOC_FLASH_NRF5_DEV_NAME, nrf5_flash_init,
 	     NULL, NULL, POST_KERNEL, CONFIG_KERNEL_INIT_PRIORITY_DEVICE);
+
+#if defined(CONFIG_SOC_FLASH_NRF5_RADIO_SYNC)
+
+static void time_slot_callback_work(u32_t ticks_at_expire, u32_t remainder,
+		u16_t lazy,
+		void *context)
+{
+	struct flash_op_desc *op_desc;
+	u8_t instance_index;
+	u8_t ticker_id;
+	int result;
+	u32_t err;
+
+	err = ll_radio_state_is_idle();
+	__ASSERT(!err, "Radio is on during flash operation.\n");
+
+	op_desc = context;
+
+	if (op_desc->handler(op_desc->context)
+		== FLASH_OP_DONE) {
```

Comment:
This looks like an unnecessary line split. Even if the split would be justified the style convention is to leave the operator (in this case ==) on the first line.